link/kprobe_multi_test.go

```diff
@@ -106,9 +106,10 @@ func TestKprobeMultiProgramCall(t *testing.T) {
 
 	// Trigger ebpf program call.
 	unix.Getpid()
+	unix.Gettid()
 
 	// Assert that the value at index 0 has been updated to 1.
```

Comment:
Can you remove the comment? It's outdated and doesn't add much.

Reply:
ok